src/DssDeploy.sol

```diff
@@ -320,7 +321,7 @@ contract DssDeploy is DSAuth {
         vow.rely(address(dog));
     }
 
-    function deployShutdown(address gov, address pit, uint256 min) public auth {
+    function deployShutdown() public auth {
```

Comment:
Perhaps we should just call this `deployEnd` now.

Reply:
sounds good

Reply:
Done.